packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/BlocksToolbar.tsx

```diff
@@ -431,6 +437,21 @@ const ListButton = ({ block, format }: ListButtonProps) => {
     }
   };
 
+  if (location === 'menu') {
+    const Icon = block.icon;
+
+    return (
+      <StyledMenuItem
+        onClick={() => toggleList(format)}
```

Comment:
Heads up Menu.Item should use onSelect. onClick will still fire with disabled=true